package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyux/phone-field",
-  "version": "4.5.0",
+  "version": "4.6.0",
```

Comment:
We update the package version in a separate PR; do you mind removing this and rerunning `npm install` (to remove the package-lock.json change)?

Reply:
Done.